frontend/css/main.sass

```diff
@@ -167,3 +170,6 @@ textarea, input[type=text], input[type=password]
     margin-bottom: 20px;
     a 
       margin-left: 10px;
+  img
```

Comment:
could you make a css class for the header image instead of applying the styles to all img tags?